src/reader/ArticleReader.js

```diff
@@ -124,7 +107,13 @@ export default function ArticleReader({ api, teacherArticleID }) {
   };
 
   const handleScroll = () => {
-    let ratio = getScrollRatio();
+    let bottomRowHeight = document.getElementById("bottomRow");
+    if (!bottomRowHeight) {
+      bottomRowHeight = 450; // 450 Is a default in case we can't acess the property
+    } else {
+      bottomRowHeight = bottomRowHeight.offsetHeight;
+    }
+    let ratio = getScrollRatio(bottomRowHeight);
```

Comment:
Somebody reading this will not be able to understand why `getScrollRatio` would require a `bottomRowHeight`. Might be solved by a renaming to something like `pageEndOffset`, or alternatively forcing the introduction of a *named parameter pattern*, something like `getScrollRatio({offset=bottomRowHeight})`. Implemented as seen here: https://byby.dev/js-named-parameters? This last solution is clearly a bit of overkill though :)

Reply:
I added a comment which seeks to address this - what do you think? It might be a bit overkill. I also updated the comment in the getScrollLocation to explain what the parameter is for. Otherwise, I can try to follow the named parameters.

Reply:
If you think it's fun to implement named parameters do it. Otherwise the new parameter at least makes it very clear what the parameter is I think!

Reply:
I will give a try to implement it tomorrow morning!